### ISSUE TITLE
Android screenshot after QR scan

### ISSUE THREAD

**Issue by parnanzio** [appcelerator-modules/ti.barcode#97]
I'm using this module on both iOS and Android.
Only on Android, when I detect a QR code the module act as a camera shot and shows a screenshot of the QR code; after about 1s it disappear.
On iOS it doesn't work this way.

Is it possibile to set Android behaviour like iOS?

**Comment by mandelmonkey** [comment]
yes its kind of looks bad from a UI point of view also it shows yellow dots when scanning, anyway to remove these two things?

**Comment by m1ga** [comment]
These should be the places which needs to be changed (e.g. add a parameter): 
https://github.com/appcelerator-modules/ti.barcode/blob/aa7ffe059df720d2d079d0e5494b365b7af7441c/android/src/com/google/zxing/client/android/ViewfinderView.java#L102
https://github.com/appcelerator-modules/ti.barcode/blob/aa7ffe059df720d2d079d0e5494b365b7af7441c/android/src/com/google/zxing/client/android/ViewfinderView.java#L183
in case someone wants to go ahead and create a PR
